Libs/Gtk-3.0/Classes/IMContextSimple.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Gtk
+{
+    public partial class IMContextSimple
+    {
+        public static IMContextSimple New()
+            => new IMContextSimple(Native.IMContextSimple.Instance.Methods.New(), false);
```

Comment:
According to documentation we own the reference returned by the new method: https://docs.gtk.org/gtk3/ctor.IMContextSimple.new.html

So `false` -> `true`?